api/resources_portal/models/attachment.py

```diff
@@ -34,6 +34,14 @@ class Meta:
     s3_bucket = models.CharField(max_length=255, blank=True, null=True)
     s3_key = models.CharField(max_length=255, blank=True, null=True)
 
+    owned_by_org = models.ForeignKey(
+        "Organization",
+        blank=False,
+        null=False,
```

Comment:
Why is `null=False` for this field? Don't we want users to be able to upload attachments for their requests. What org are those owned by?

Reply:
Good point. Changed to True.